.gitignore

```diff
@@ -109,3 +109,6 @@ packages/sql-indexer/knexfile.dev.js
 # compiled js
 packages/helpers/lib
 packages/common-scripts/lib
+
+# knex config
+packages/sql-indexer/knexfile.ts
```

Comment:
Please handle proper end of line in a file.

Reply:
Oops, I'll fix it.